smoke-test.sh

```diff
@@ -26,41 +26,59 @@ cp -r "$TEST_DIR/remote" "$TEST_DIR/vendored" "$EXAMPLES_DIR"
 touch "$EXAMPLES_DIR/BUILD"
 
 # Set up the new WORKSPACE file
-cat > "$EXAMPLES_DIR/WORKSPACE" << EOF
-workspace(name = "examples")
+cat > "$EXAMPLES_DIR/WORKSPACE.bazel" << EOF
+workspace(name = "cargo_raze_examples")
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-git_repository(
-    name = "io_bazel_rules_rust",
-    commit = "fda9a1ce6482973adfda022cadbfa6b300e269c3",
-    remote = "https://github.com/bazelbuild/rules_rust.git",
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+    ],
 )
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+
+bazel_skylib_workspace()
+
 http_archive(
-    name = "bazel_skylib",
-    sha256 = "eb5c57e4c12e68c0c20bc774bfbc60a568e800d025557bc4ea022c6479acc867",
-    strip_prefix = "bazel-skylib-0.6.0",
-    url = "https://github.com/bazelbuild/bazel-skylib/archive/0.6.0.tar.gz",
+    name = "io_bazel_rules_rust",
+    sha256 = "5ed804fcd10a506a5b8e9e59bc6b3b7f43bc30c87ce4670e6f78df43604894fd",
+    strip_prefix = "rules_rust-fdf9655ba95616e0314b4e0ebab40bb0c5fe005c",
+    urls = [
+        # Master branch as of 2020-07-27
+        "https://github.com/bazelbuild/rules_rust/archive/fdf9655ba95616e0314b4e0ebab40bb0c5fe005c.tar.gz",
+    ],
 )
-load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
-bazel_version(name = "bazel_version")
 
 load("@io_bazel_rules_rust//rust:repositories.bzl", "rust_repositories")
+
 rust_repositories()
 
+load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
+
+bazel_version(name = "bazel_version")
+
 EOF
 
 for ex in $(find $TEST_DIR/remote -maxdepth 1 -type d | tail -n+2); do
     name="$(basename "$ex")"
-    cat >> "$EXAMPLES_DIR/WORKSPACE" << EOF
+    cat >> "$EXAMPLES_DIR/WORKSPACE.bazel" << EOF
 load("//remote/${name}/cargo:crates.bzl", "${name}_fetch_remote_crates")
+
 ${name}_fetch_remote_crates()
 
 EOF
 done
 
+# Define a Bazel version file
+cat > "$EXAMPLES_DIR/.bazelversion" << EOF
```

Comment:
This isn't used by CI, so I have a feeling this will mostly cause confusion if it isn't kept up to date with the version set here https://github.com/google/cargo-raze/blob/eac33e62a5895fdd73630ae2edd2b456b2f72595/.travis.yml#L17

If this change is gonna be made, it might make sense to have `.travis.yml` do something like `V="$(cat examples/.bazelversion)"` instead

Reply:
Good call. I updated this but also just added a `.bazelversion` file to the root of the repo to be the single source of truth. This should hopefully be clearer for everyone.